File: Ruby/27.rb
print "Enter the number to check if it's a palindrome!:\t"
num = $stdin.gets.chomp.to_s
num = num.split(//)
num2 = num.reverse
pos = 0
flag = 1

for i in num
  if num[pos] != num2[pos] 
    flag = 0
  end
  pos = pos+1
end

if(flag == 1)
  print "\nThe entered number is a palindrome"

else
  print "\nThe entered number is not a palindrome"
end

File: Python/27.py
# Python program to check if a number is Palindrome or not

n=int(input("Enter number:"))
num=n
rev=0
while(n>0):
    dig=n%10
    rev=rev*10+dig
    n=n//10
if(num==rev):
    print("The number is a palindrome")
else:
    print("The number is not a palindrome!")
    
    
# Enter number:12199
# The number is not a palindrome
